docker/requirements.txt

```diff
@@ -32,6 +32,8 @@ dask[complete]==2.8.1
 distributed==2.8.1
 xarray
 tensorflow
+# REMOVE ME AFTER TESTING
```

Comment:
I think just adding comment line without injecting an extra lib is enough to cause rebuild